extensions/rocksdb-repos/DatabaseContentRepository.h

```diff
@@ -69,12 +72,23 @@ class DatabaseContentRepository : public core::ContentRepository {
 
   void clearOrphans() override;
 
+  void start() override;
+  void stop() override;
+
+  // test-only utility
+  void test_deinitialize();
```

Comment:
It may be better to create a TestDatabaseContentRepository in the test codebase that inherits from this and implement the test only functions there. Also please use camelCase here.

Reply:
done